pkg/edge/chromium.go

```
@@ -691,12 +612,12 @@ func (e *Chromium) PutIsSwipeNavigationEnabled(enabled bool) error {
 	}
 	webview2Settings, err := e.webview.GetSettings()
 	if err != nil {
-		return err
+		return nil
 	}
 	webview2Settings6 := webview2Settings.GetICoreWebView2Settings6()
 	err = webview2Settings6.PutIsSwipeNavigationEnabled(enabled)
 	if !errors.Is(err, windows.DS_S_SUCCESS) {
-		return err
+		return nil
```

If we're going to ignore the error here shouldn't we just remove the check?